usaon_benefit_tool/templates/project/edit.html

```diff
@@ -1,14 +1,15 @@
-{% extends 'survey/base.html' %}
+{% extends 'project/base.html' %}
 {% from 'bootstrap5/utils.html' import render_icon %}
 {% from 'macros/javascript_highcharts.j2' import display_sankey %}
 
 {% block content %}
   {{super()}}
 
   <h3>Description</h3>
-  <p>{{survey.description}}</p>
+  <p>{{project.description}}</p>
 
 
-  {{ display_sankey(sankey_series, survey_title=survey.title)}}
+  I am the edit interface.
```

Comment:
TODO:

- [ ] Remove :laughing:
- [ ] Actually make the overview and edit interfaces different; right now all the prototyping work is being done on the overview interface.